src/CoreWCF.ConfigurationManager/tests/MachineConfigTransitiveDependencyTests.cs

```diff
@@ -0,0 +1,47 @@
+using System;
+using System.IO;
+using Xunit;
+
+namespace CoreWCF.ConfigurationManager.Tests
```

Comment:
I'm wondering if the unit tests should instead orchestrate création of a csproj,
installation of CoreWCF (as direct and transitive dependency) and assertion. The unit tests of the templates are doing something similar.

Reply:
I did manual testing to validate the scenario to make sure it works. I'll open an issue to track creating a better integration test.